draft-aegis-mimi-arch-latest.md

```diff
@@ -189,37 +188,99 @@ struct {
 
 ### X.509 Identifiers
 
-Calculating a client's identifier is done by iterating through a subset of X.509
-subject fields found within the client's certificate chain. The range of
-certificates in the chain to reference as part of the calculation is defined as
-starting with `start` values from the leaf and ending with
-`end` values from the leaf. A single unique identifier is calculated by
-hashing the Common Name value found in each certificate within the reference
-range using the current group's ciphersuite hash function.
+Identifiers (e.g. for clients) are derived by iterating through a subset of X.509 subject Common Name fields found within the client's certificate chain. Common Name fields MUST not contain any of the 5 special characters "@", "/", "#", "$" and ":". The range of certificates in the chain to used in the derivation is defined as starting with `start` values from the leaf and ending with `end` values from the leaf.
```

Comment:
X.509 standard allows the full Unicode set to work here. I feel like it's overly restrictive to remove characters when applications might have a valid use for them

Reply:
The thought here is that those characters are reserved in the URI scheme laid out in Rohan's identity doc. allowing them here would make the URI's ambiguous.